src/me/StevenLawson/TotalFreedomMod/TFM_ServerInterface.java

```diff
@@ -241,4 +247,41 @@ public static void handlePlayerLogin(PlayerLoginEvent event)
             }
         }
     }
+    
+     //Please note that with titles, you must ALWAYS send the title first, and the subtitle second.
+    public static void sendTitle(Player player, String message, int fadein, int stay, int fadeout)
+    {
+        try
+        {
+            CraftPlayer craftplayer = (CraftPlayer) player;
+            PlayerConnection connection = craftplayer.getHandle().playerConnection;
+            String finalmessage = message.replaceAll("&", "§");
+            IChatBaseComponent chatTitle = ChatSerializer.a("{\"text\": \"" + finalmessage + "\"}");
+            PacketPlayOutTitle title = new PacketPlayOutTitle(EnumTitleAction.TITLE, chatTitle, fadein, stay, fadeout);
+            connection.sendPacket(title);
+        }
+        catch (Exception e)
+        {
+            TFM_Log.severe(e.getLocalizedMessage());
+            TFM_Log.severe("Title packet send failed. Your server jar is likely outdated.");
```

Comment:
For standardisation purposes, the "info" message should be sent before the stack trace. And it should describe only the problem that occurred, not the possible cause.

In this case, something along the lines of `"Could not send title packet to player " + player.getName()` should suffice.